src/overlays/actors/ovl_Demo_Ec/z_demo_ec.c

```diff
@@ -9,7 +10,1333 @@ void DemoEc_Destroy(Actor* thisx, GlobalContext* globalCtx);
+    CsCmdActorAction* npcAction = DemoEc_GetNpcAction(globalCtx, actionIndex);
+
+    if (npcAction != NULL) {
+        this->actor.posRot.pos.x = npcAction->startPos.x;
+        this->actor.posRot.pos.y = npcAction->startPos.y;
+        this->actor.posRot.pos.z = npcAction->startPos.z;
+
+        this->actor.posRot.rot.y = this->actor.shape.rot.y = npcAction->rot.y;
+    }
+}
+
+void DemoEc_InitIngo(DemoEc* this, GlobalContext* globalCtx) {
+    DemoEc_UseDrawObject(this, globalCtx);
+    DemoEc_InitSkelAnime(this, globalCtx, &D_06013B88);
+    DemoEc_UseAnimationObject(this, globalCtx);
+    DemoEc_ChangeAnimation(this, &D_060048F4, 0, 0.0f, false);
+    func_8096D64C(this, globalCtx);
+    ActorShape_Init(&this->actor.shape, 0.0f, ActorShadow_DrawFunc_Circle, 30.0f);
+    this->updateMode = EC_UPDATE_INGO;
+    this->drawConfig = EC_DRAW_INGO;
+}
+
+void DemoEc_UpdateIngo(DemoEc* this, GlobalContext* globalCtx) {
+    DemoEc_UpdateSkelAnime(this);
+    func_8096D594(this, globalCtx);
+    DemoEc_UpdateBgFlags(this, globalCtx);
+}
+
+void DemoEc_DrawIngo(DemoEc* this, GlobalContext* globalCtx) {
+    DemoEc_DrawSkeleton(this, globalCtx, &D_06004390, &D_06004350, 0, 0);
+}
+
+void DemoEc_InitTalon(DemoEc* this, GlobalContext* globalCtx) {
+    DemoEc_UseDrawObject(this, globalCtx);
+    DemoEc_InitSkelAnime(this, globalCtx, &D_0600B7B8);
+    DemoEc_UseAnimationObject(this, globalCtx);
+    DemoEc_ChangeAnimation(this, &D_0600BD38, 0, 0.0f, false);
+    func_8096D64C(this, globalCtx);
+    ActorShape_Init(&this->actor.shape, 0.0f, ActorShadow_DrawFunc_Circle, 30.0f);
+    this->updateMode = EC_UPDATE_TALON;
+    this->drawConfig = EC_DRAW_TALON;
+}
+
+void DemoEc_UpdateTalon(DemoEc* this, GlobalContext* globalCtx) {
+    DemoEc_UpdateSkelAnime(this);
+    func_8096D594(this, globalCtx);
+    DemoEc_UpdateBgFlags(this, globalCtx);
+}
+
+void DemoEc_DrawTalon(DemoEc* this, GlobalContext* globalCtx) {
+    DemoEc_DrawSkeleton(this, globalCtx, &D_060076C0, &D_06007AC0, NULL, NULL);
+}
+
+void DemoEc_InitWindmillMan(DemoEc* this, GlobalContext* globalCtx) {
+    DemoEc_UseDrawObject(this, globalCtx);
+    DemoEc_InitSkelAnime(this, globalCtx, &D_06006C90);
+    DemoEc_UseAnimationObject(this, globalCtx);
+    DemoEc_ChangeAnimation(this, &D_0600196C, 0, 0.0f, false);
+    func_8096D5D4(this, globalCtx);
+    ActorShape_Init(&this->actor.shape, 0.0f, ActorShadow_DrawFunc_Circle, 30.0f);
+    this->updateMode = EC_UPDATE_WINDMILL_MAN;
+    this->drawConfig = EC_DRAW_WINDMILL_MAN;
+}
+
+void DemoEc_UpdateWindmillMan(DemoEc* this, GlobalContext* globalCtx) {
+    DemoEc_UpdateSkelAnime(this);
+    func_8096D594(this, globalCtx);
+    DemoEc_UpdateBgFlags(this, globalCtx);
+}
+
+void DemoEc_DrawWindmillMan(DemoEc* this, GlobalContext* globalCtx) {
+    DemoEc_DrawSkeleton(this, globalCtx, &D_06005F20, &D_06006920, NULL, NULL);
+}
+
+void DemEc_InitKokiriBoy(DemoEc* this, GlobalContext* globalCtx) {
```

Comment:
```suggestion
void DemoEc_InitKokiriBoy(DemoEc* this, GlobalContext* globalCtx) {
```

Reply:
sorry what is this change showing it looks the same to me

Reply:
nvm